CompetitionTask/StepDefinitions/MarsEducationCertificationsTestsStepDefinitions.cs

```diff
@@ -0,0 +1,269 @@
+using Mars_Education_Certifications.Pages;
+using MarsEducationCertificationsSpecflow.Pages;
+using MarsEducationCertificationsSpecflow.Utilities;
+using OpenQA.Selenium;
+using AventStack.ExtentReports.Gherkin.Model;
+using MarsEducationCertificationsSpecflow.JSONDataObject;
+using MarsEducationCertificationsTests.JSONDataObject;
+using System.Diagnostics.Metrics;
+using MarsEduCertAutomation.Utilities;
+using NUnit.Framework;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using System.Collections.Generic;
+using MarsEduCertAutomation.Utility;
+using NUnit.Framework.Interfaces;
+
+
+namespace MarsEducationCertificationsSpecflow.StepDefinitions
```

Comment:
also i would not call this step definition but tests

Reply:
Resolved